documentation/src/main/asciidoc/topics/proc_cli_xsite_sites.adoc

```diff
@@ -0,0 +1,25 @@
+[id='manage_sites-{context}']
+= Bringing Sites Offline and Online
+Take sites offline manually and bring sites back online.
+
+.Procedure
+
+* Check if sites are online or offline with the `site status` command:
++
+----
+//containers/default]> site status -s LON
```

Comment:
this needs to be updated :( 
1. `The option -s is unknown.`
2. `Option: --cache is required for this command.`

usage example:
` site status --cache=___script_cache --site=NYC`

`--site` is optional. if not set, it will print all backups configured.

Reply:
thanks, I think the `-s` option was an older one that got stuck in there. `--site` isn't always optional from what I can tell, right? in the case of `site status` it is optional. but in the case of `site cancel-receive-state` you need to provide the remote site. isn't that so?